diff --git a/node_modules/jimp/jimp.d.ts b/node_modules/jimp/jimp.d.ts
index 2a70281..e9e1ba0 100644
--- a/node_modules/jimp/jimp.d.ts
+++ b/node_modules/jimp/jimp.d.ts
@@ -126,11 +126,11 @@ interface Jimp {
   quality(n: number, cb?: ImageCallback): this;
   getBase64(mime: string, cb: GenericCallback<string, any, this>): this;
   getBase64Async(mime: string): Promise<string>;
-  hash(cb?: GenericCallback<string, any, this>): this;
+  hash(cb?: GenericCallback<string, any, this>): string;
   hash(
     base: number | null | undefined,
     cb?: GenericCallback<string, any, this>
-  ): this;
+  ): string;
   getBuffer(mime: string, cb: GenericCallback<Buffer>): this;
   getBufferAsync(mime: string): Promise<Buffer>;
   getPixelIndex(
